File: src/androidTest/java/pt/joaomneto/titancompanion/TestCOTS.java
package pt.joaomneto.titancompanion;


import android.support.test.filters.LargeTest;
import android.support.test.runner.AndroidJUnit4;

import org.junit.runner.RunWith;

import pt.joaomneto.titancompanion.consts.FightingFantasyGamebook;

import static pt.joaomneto.titancompanion.consts.FightingFantasyGamebook.CRYPT_OF_THE_SORCERER;

@LargeTest
@RunWith(AndroidJUnit4.class)
public class TestCOTS extends TestST {

    @Override
    protected FightingFantasyGamebook getGamebook() {
        return CRYPT_OF_THE_SORCERER;
    }


}

File: src/androidTest/java/pt/joaomneto/titancompanion/phase1/TestCOTS.kt
package pt.joaomneto.titancompanion.phase1

import androidx.test.filters.LargeTest
import androidx.test.ext.junit.runners.AndroidJUnit4
import org.junit.runner.RunWith
import pt.joaomneto.titancompanion.consts.FightingFantasyGamebook.CRYPT_OF_THE_SORCERER

@LargeTest
@RunWith(AndroidJUnit4::class)
class TestCOTS : TestST() {

    override val gamebook = CRYPT_OF_THE_SORCERER
}
